apps/wallet/src/ui/app/pages/home/nft-transfer/TransferNFTForm.tsx

```diff
@@ -48,7 +49,12 @@ export function TransferNFTForm({ objectId }: { objectId: string }) {
         },
         onSuccess: (response) => {
             queryClient.invalidateQueries(['object', objectId]);
-            queryClient.invalidateQueries(['objects-owned']);
+            queryClient.invalidateQueries([
```

Comment:
thinking if just clearing all `get-owned-objects` is better. We could send objects to another address of the same wallet as well